service/src/main/java/org/broadinstitute/listener/config/CorsSupportProperties.java

```diff
@@ -0,0 +1,3 @@
+package org.broadinstitute.listener.config;
+
+public record CorsSupportProperties(String preflightMethods) {}
```

Comment:
We should update the README with documentation for this property - its purpose and default value.